test/converters/xcresult/testsummariesplist.go

```diff
@@ -24,7 +24,8 @@ func collapseSubtestTree(data Subtests) (tests Subtests) {
 }
 
 // Tests returns the collapsed tree of tests
-func (summaryPlist TestSummaryPlist) Tests() map[string]Subtests {
+func (summaryPlist TestSummaryPlist) Tests() ([]string, map[string]Subtests) {
+	keyOrder := []string{}
```

Comment:
NOTE: better to declare a slice using `var keyOrder []string`: https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#declaring-empty-slices